contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -516,6 +517,7 @@ sp_describe_first_result_set_internal(PG_FUNCTION_ARGS)
 				pfree(query);
 				pfree(sp_describe_first_result_set_view_name);
 				SPI_finish();
+				RESUME_INTERRUPTS();
```

Comment:
we should resume if SPI_EXECUTE fails, elog(ERROR) will throw and error

Reply:
DOne